### ISSUE TITLE
🪲 Add and remove accept text without quotes from level 12 onwards

### ISSUE THREAD

**Issue by jpelay** [hedyorg/hedy#5692]
**Describe the bug**
On level 12 we replace almost all instances of text without quotes to require them, this is true for assignments, list assignments, comparisons, etc. There's, however, still a case where text without quotes is still allowed: the `add` and `remove` commands. These two commands are defined as follows:

```
add: _ADD_LIST (call | some_spaces) _TO_LIST var_access
remove: _REMOVE (call | some_spaces) _FROM var_access

some_spaces : /([^\r\n!?. ]+)/ (_SPACE /([^\r\n!?. ]+)/)* -> text
```

I think quotes should be required around these commands as well.

**Comment by jpelay** [comment]
What do you think @boryanagoncharenko?

**Comment by boryanagoncharenko** [comment]
This is the issue I mentioned to you during our last meeting! I already addressed it in #5676. Not only it feels odd that we allow it, it is actually hard to support it because we need different argument processing. Here is how the line looks in the PR:
```
add: _ADD_LIST (atom | list_access | call) _TO_LIST var_access
remove: _REMOVE (atom | list_access | call) _FROM var_access
```
Btw, I removed the rule `some_spaces` altogether because it seemed like [something was off](https://github.com/hedyorg/hedy/blob/9c9ecadad69081ef90b00ff9acdc5dde017e494a/grammars/level3-Additions.lark#L16) with it anyways. I adjusted the spacing of the terminals and it everything worked without it!

**Comment by jpelay** [comment]
> This is the issue I mentioned to you during our last meeting! I already addressed it in #5676. Not only it feels odd that we allow it, it is actually hard to support it because we need different argument processing. Here is how the line looks in the PR:

Oh yes you're right! 


> Btw, I removed the rule `some_spaces` altogether because it seemed like [something was off](https://github.com/hedyorg/hedy/blob/9c9ecadad69081ef90b00ff9acdc5dde017e494a/grammars/level3-Additions.lark#L16) with it anyways. I adjusted the spacing of the terminals and it everything worked without it!

It always seemed funky to me!

**Comment by boryanagoncharenko** [comment]
Since #5676 is merged to main, I am closing this. However, if you feel there is more to this issue, please reopen it.
